Dockerfile

```diff
@@ -25,12 +34,13 @@ ADD build-config.json /opt/opentripplanner/build/
 ADD otp-config.json /opt/opentripplanner/build/
 ADD $OSM_PBF_URL /opt/opentripplanner/build/
 ADD $GTFS_URL /opt/opentripplanner/build/gtfs.zip
+RUN cp mfdz.bb.gtfs.zip /opt/opentripplanner/build/gtfs-carpool.zip
+ADD $GTFS_FLEXFEED_URL /opt/opentripplanner/build/
 ADD dgm/* /opt/opentripplanner/build/
```

Comment:
This can be moved up, it well change rarely compared to e.g. GTFS feeds.

Reply:
Indeed